crypto/ocsp/ocsp_http.c

```diff
@@ -520,3 +519,14 @@ BIO *OCSP_REQ_CTX_get0_mem_bio(OCSP_REQ_CTX *rctx)
 {
     return rctx->mem;
 }
+
+void OCSP_set_max_response_length(OCSP_REQ_CTX *rctx, unsigned long len)
```

Comment:
What is max_resp_len used for? I expect that after changing this size some buffer needs to be re-allocated

Reply:
This is used when anticipating the maximum size of the HTTP OCSP response sent back from the OCSP responder in `OCSP_REQ_CTX_nbio`.
* https://github.com/aws/aws-lc/blob/ocsp-request/crypto/ocsp/ocsp_http.c#L314

This function should be set before the `OCSP_REQ_CTX` is sent out with `OCSP_REQ_CTX_nbio` (I'll add a comment for that). The variable is only used as a max response length check, so no memory needs to be reallocated here.